scylla/src/transport/locator/test.rs

```diff
@@ -245,6 +257,7 @@ fn test_simple_strategy_replicas(locator: &ReplicaLocator) {
                 replication_factor: 3,
             },
             None,
+            &TableSpec::borrowed("ks", "invalid"),
```

Comment:
Please make this a named constant and use in all places.

Reply:
What is the intention here? I guess that the table is intended as invalid, what about keyspace? `"ks"` tells me nothing.

Reply:
Constant was already declared actually. Changed those `borrowed` calls to use the constant. Changed the constant to `TableSpec::borrowed("invalid", "invalid")`